variables.go

```diff
@@ -123,10 +123,60 @@ func (e *Executor) compiledTask(call taskfile.Call, evaluateShVars bool) (*taskf
 			if cmd == nil {
 				continue
 			}
+			if cmd.ForEach != nil {
+				var list []string
+				// Get the list from the explicit foreach list
+				if cmd.ForEach.List != nil && len(cmd.ForEach.List) > 0 {
+					list = cmd.ForEach.List
+				}
+				// Get the list from the task sources
+				if cmd.ForEach.From == "source" {
+					list, err = fingerprint.Globs(new.Dir, new.Sources)
+					if err != nil {
+						return nil, err
+					}
+				}
+				// Get the list from a variable and split it up
+				if cmd.ForEach.Var != "" {
+					if vars != nil {
+						v := vars.Get(cmd.ForEach.Var)
+						if cmd.ForEach.Split != "" {
+							list = strings.Split(v.Static, cmd.ForEach.Split)
+						} else {
+							list = strings.Fields(v.Static)
+						}
+					}
+				}
+				// Name the iterator variable
+				var as string
+				if cmd.ForEach.As != "" {
+					as = cmd.ForEach.As
+				} else {
+					as = "LOOP_VALUE"
```

Comment:
What about `ITEM`? It's shorter and more coherent with the word "list" as we're using in the documentation for this.

Reply:
I did spend a while thinking about this (naming is hard) and `ITEM` was my first choice. I ended up with `LOOP_VALUE` because I wanted something with a bit of context - i.e. that suggests that the value is coming from the loop rather than some random user-set variable.

However, if you think that `ITEM` is contextual enough, then I have no strong feeling against it.

Reply:
I think `ITEM` is good enough, and a bit cleaner. Good thing that you made it overridable.